hedera-node/hedera-app/src/xtest/java/contract/WipeXTest.java

```diff
@@ -158,6 +175,77 @@ protected void doScenarioOperations() {
                         Bytes.wrap(ReturnTypes.encodedRc(INVALID_ACCOUNT_ID).array()),
                         output,
                         "Expected INVALID_ACCOUNT_ID when trying to execute with invalid account address (V2)"));
```

Comment:
Can we exclude the operations for matching Mono Behaviours in another method like `assertWipeOperationsMatchMonoBehaviour`? Having them with this comment in the middle "// @Future remove to revert #9272 after modularization is completed," can cause confusion and we might mistake that they include the key verification. Hence, we can move them to another method and call it as the last operations.

We can also document it at the top of the class as something like "Via {@code assertWipeOperationsMatchMonoBehaviour} ensure wipe operations in modularization match mono behaviour."

Reply:
I believe that we will have to change the whole test scenario as now we expect `SUCCESS + Error message` with all the tests in this case, so I don't think we need to externalise the specific "mono behaviour" ones.